index.html

```diff
@@ -4693,6 +4693,100 @@ <h2>
 </p>
     </section>
 
+    <section>
+      <h2>Verification Method Revocation</h2>
+  
+      <p>Verification method revocation is a reactive security measure.</p>
+
+      <p>
+        Verification method revocation applies only to the current or latest
+        verison of a DID Document.
+      </p>
+  
+      <p>
+        If a verification method is no longer exclusively accessible to the
+        controller or parties trusted to act on behalf of the controller, it
+        should be revoked immediatly to reduce the risk of masquerading, theft
+        and fraud.
```

Comment:
```suggestion
If a verification method is no longer exclusively accessible to the
controller or parties trusted to act on behalf of the controller, it
should be immediately revoked to reduce the risk of masquerading, theft,
and fraud.
```

Reply:
I am resolving this, since I cannot apply it.